client/http/api.go

```diff
@@ -182,11 +181,6 @@ func SchedulerByName(name string) string {
 	return fmt.Sprintf("%s/%s", Schedulers, name)
 }
 
-// ScatterRangeSchedulerWithName returns the scatter range scheduler API with name parameter.
-func ScatterRangeSchedulerWithName(name string) string {
```

Comment:
It's a break change.

Reply:
It is not used🤔

Reply:
TiDB uses it.

Reply:
I have not found it in TiDB. And where is the service interface implement in PD server? 🤔

Reply:
https://github.com/pingcap/tidb/blob/2a3352c45dd0f8dd5102adb92879bbfa964e7f5f/pkg/server/handler/tikvhandler/tikv_handler.go#L1252